VMT/KCSA.md

```diff
@@ -4,10 +4,11 @@ This page lists all previously published Kata Containers Security Advisories (KC
 
 This table is in reverse date order.
 
-| KCSA                                             | Description                  |
-| ------------------------------------------------ | ---------------------------- |
-| [KCSA-CVE-2019-5736](KCSA/KCSA-CVE-2019-5736.md) | runc container breakout      |
-| [KCSA-CVE-2020-2024](KCSA/KCSA-CVE-2020-2024.md) | improper link resolution vulnerability |
-| [KCSA-CVE-2020-2025](KCSA/KCSA-CVE-2020-2025.md) | Cloud Hypervisor guest image persists vulnerability |
-| [KCSA-CVE-2020-2023](KCSA/KCSA-CVE-2020-2023.md) | Execution with Unnecessary Privileges |
-| [KCSA-CVE-2020-2026](KCSA/KCSA-CVE-2020-2026.md) | Improper Link Resolution Before File Access |
+| KCSA                                               | Description                  |
+| -------------------------------------------------- | ---------------------------- |
+| [KCSA-CVE-2020-28914](KCSA/KCSA-CVE-2020-28914.md) | Improper file permissions for read-only volumes |
+| [KCSA-CVE-2020-2023] (KCSA/KCSA-CVE-2020-2023.md)  | Execution with Unnecessary Privileges |
+| [KCSA-CVE-2020-2026] (KCSA/KCSA-CVE-2020-2026.md)  | Improper Link Resolution Before File Access |
+| [KCSA-CVE-2020-2024] (KCSA/KCSA-CVE-2020-2024.md)  | improper link resolution vulnerability |
+| [KCSA-CVE-2020-2025] (KCSA/KCSA-CVE-2020-2025.md)  | Cloud Hypervisor guest image persists vulnerability |
+| [KCSA-CVE-2019-5736] (KCSA/KCSA-CVE-2019-5736.md)  | runc container breakout      |
```

Comment:
You need to remove the space between the closing square bracket and the opening parenthesis to unbreak these links.